content/arduino-cloud/03.tutorials/document-projects-through-webeditor/classroom_project.md

```diff
@@ -0,0 +1,103 @@
+---
+title: 'Creating Projects using the Web Editor'
+description: 'An article on how a teacher can use the web editor '
+tags: 
+  - article
+author: 'Lenard George Swamy'
+---
+
+## Introduction 
+
+Many of you might have an interesting idea for your project but struggle with documentation and sharing it with your class. In this article you will learn how to use the web editor to document a project. We will illustrate the process with an example project  
+
+***Note: We recommend that you read the article (Getting started with the Web Editor)(/getting-started/getting-started-web-editor) before you proceed with this tutorial.***
+
+## The Essentials 
+Every arduino project has 3 essential elements that will make your teaching effortless. These are
```

Comment:
```suggestion
Every Arduino project has 3 essential elements that will make your teaching effortless. These are
```

Reply:
It seems like there is missing a section after the bullet point list to explain why these are the 3 essential elements. Perhaps you could elaborate?